src/operations.cc

```diff
@@ -406,7 +406,9 @@ namespace sharp {
 
     // we need to smooth the image, subtract the background from every pixel, take
     // the absolute value of the difference, then threshold
-    VImage mask = (image.median(3) - background).abs() > threshold;
+    // TODO: Not sure if this calculation supports tolerance. Or should we do this?:
+    // VImage mask = (image.median(3) - background).abs() > (MaximumImageAlpha(image.interpretation()) / 100 * tolerance);
```

Comment:
I think I'd have this as `max * tolerance / 100`. I guess this should be uncommented too?